### PR TITLE
Adding more analysis tools

### PR DESCRIPTION
**Pull request by tsenapathi**

### CODE REVIEW

File: tools/bio3d/visualize_pc.xml [hunk 1]
```diff
@@ -0,0 +1,141 @@
+<tool id="bio3d_pca_visualize" name="PCA visualization" version="@VERSION@">
+    <description>of principle components of motions</description>
```
Comment (bgruening) [inline tools/bio3d/visualize_pc.xml]: can/should be add "protein"/"structure" to the description?

Comment (tsenapathi) [reply tools/bio3d/visualize_pc.xml]: Thanks @bgruening I will add this

**Comment by bgruening** [comment]
Merci!

**Comment by tsenapathi** [comment]
Thank you @bgruening

**Pull request merged**